utils/src/server/rpc_module.rs

```diff
@@ -421,11 +428,12 @@ impl<Context: Send + Sync + 'static> RpcModule<Context> {
 		F: Fn(Params, &Context) -> Result<R, Error> + Send + Sync + 'static,
 	{
 		let ctx = self.ctx.clone();
+		let max_call_size = self.max_call_size;
 		let callback = self.methods.verify_and_insert(
 			method_name,
 			MethodCallback::new_sync(Arc::new(move |id, params, tx, _| {
```

Comment:
I think it might be better if we pass the `max_call_size` (shouldn't it be `max_response_size`?) as a param to closures here, so it can be set by either server when calling the method. This would need piping it through on every `execute`, but I reckon that's better than configuring it globally on the module itself.

Reply:
+1 for `max_response_size`

Reply:
thanks yeah indeed much cleaner